master/buildbot/config.py

```diff
@@ -389,7 +378,7 @@ def copy_str_param(name, alt_key=None):
             # would hide the title.
             warnings.warn('WARNING: Title is too long to be displayed. ' +
                           '"Buildbot" will be used instead.',
-                          category=DeprecatedConfigWarning)
+                          category=ConfigWarning)
```

Comment:
I agree we should have a ConfigWarning class that for warnings that are not deprecations. but it seems you omitted to add it.

Reply:
I added it back in #5353.